field/hash/hashutils.go

```diff
@@ -6,8 +6,8 @@ import (
 )
 
 // ExpandMsgXmd expands msg to a slice of lenInBytes bytes.
-// https://tools.ietf.org/html/draft-irtf-cfrg-hash-to-curve-06#section-5
-// https://tools.ietf.org/html/rfc8017#section-4.1 (I2OSP/O2ISP)
+// https://tools.ietf.org/doc/html/rfc9380#name-expand_message_xmd
```

Comment:
The links redirect to IETF home page. Maybe use https://www.rfc-editor.org/rfc/rfc9380.html instead?

Reply:
Or use https://datatracker.ietf.org/doc/html/rfc9380#name-expand_message_xmd

Reply:
sure